src/useSteps.js

```diff
@@ -18,8 +18,8 @@ const options = [
   { type: "characteristic", value: "spicy", label: "Spicy" },
   {
     type: "characteristic",
-    value: "forest berry aroma",
-    label: "Forest berry aroma",
+    value: "forest berry",
+    label: "forest berry",
```

Comment:
Make sure that labels are all capitalised:
```suggestion
    label: "Forest berry",
```

Also there should be no repetitions here, and no unused characteristics.